ApollosProject/apollos-apps
# fix: Community Feed Header

## ✏️ Description

There was a problem where the header was overlapping with the content of the "Community" tab. This PR enables the proper inset adjustments in order to make sure that everything renders correctly.

There was also a flipped boolean flag that this PR also resolves so that no Stream Provider error is thrown when enabling Stream for an individual client

## 🔬 To Test
1. make sure that `ENABLE_STREAM=true` is set in your env
2. Run the app and navigate to the Community tab

## 📸 Screenshots
| Broken | Fixed |
| --- | --- |
| ![IMG_ADD34007B249-1](https://user-images.githubusercontent.com/45076058/213807304-f3691553-6545-4fb0-bce8-41202d874e37.jpeg) | ![CleanShot 2023-01-20 at 16 14 55](https://user-images.githubusercontent.com/45076058/213807375-cbda54d5-0f4c-4087-8d1a-e8a14261edb1.png) |



## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
